server/api/user_service_profile.py

```diff
@@ -55,6 +55,11 @@ def attributes():
             user_service_profile_val = getattr(user_service_profile, claim)
             if user_service_profile_val:
                 result.setdefault(user_service_profile_saml_mapping[claim], []).extend([user_service_profile_val])
+    cos = set(map(lambda usp: usp.authorisation_group.collaboration.short_name, user_service_profiles))
+    if dict(current_app.app_config).get("generate_multiple_eppn", False):
+        eppns = list(map(lambda co: f"{user.username}@{co}.{current_app.app_config.base_scope}", cos))
+        result.setdefault("urn:mace:dir:attribute-def:eduPersonPrincipalName", []).extend(eppns)
```

Comment:
@oharsta Ik twijfel of dit (hardcoded attribute name in de code) de beste manier is om dit probleem (aanmaken van afgeleide attributen) aan te pakken, maar omdat dit de eerste keer is dat we dit tegenkomen hebben ze hier nog geen andere manieren voor .
Heb jij ene suggestie hoe het beter kan?